src/components/pages/Home/RenderHomePage.js

```diff
@@ -1,7 +1,9 @@
-import React, { useEffect } from 'react';
-import { connect } from 'react-redux';
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+function RenderHomePage() {
+  const userData = useSelector(state => state.postProfileReducer.userData);
```

Comment:
I like that you made the switch to Redux hooks. I find them more readable and easier to reason about.